src/lib/molecules/header.svelte

```diff
@@ -1,67 +1,67 @@
 <script>
   let active = false;
-
-  function myFunction() {
-    var x = document.getElementById("myLinks");
-    if (x.style.display === "block") {
-      x.style.display = "none";
-      active = false;
-    } else {
-      x.style.display = "block";
-      active = true;
-    }
-  }
 </script>
 
+<a href="/" class="logo"><img src="/vtHBO-v1.0-Logo_1.svg"  alt="Visual Thinking" /></a>
```

Comment:
Zou dit niet een betere alternatieve benaming zijn voor het logo? > "Visual Thinking Logo"

Reply:
Ja is beter! Je hebt gelijk!